.github/workflows/nightly-tests.yml

```diff
@@ -97,7 +111,7 @@ jobs:
   notify-de-slack:
     needs: [ build, jib, test-runner ]
     runs-on: ubuntu-latest
-    if: failure()
+    if: ${{ github.event_name != 'workflow_dispatch' && failure() }}
```

Comment:
Is this logic correct? If this workflow is triggered by a Jenkins job, would we want to see failures in our slack? If not, how would we be notified of failures? Maybe sending a message directly from Jenkins would be better, if that can be done.

Reply:
workflow dispatch is the manual trigger that is done via the bash script I added, so this won't be reported if it's manual trigger and fails

Reply:
> workflow dispatch is the manual trigger that is done via the bash script I added, so this won't be reported if it's manual trigger and fails

So to be clear; if this is run by a workflow dispatch (which is what Jenkins will use) then errors _won't_ be reported to slack? Why wouldn't we want to see errors caused by the Jenkins job trigger? Or are those errors reported to our slack via some other system?

Reply:
this was just done with the script in mind for locally testing the github action, if it will conflict with jenkins then I can change it back

Reply:
OK; I was asking because I don't know how errors are reported from the Jenkins jobs that run alpha and staging tests. I assume that they don't report status in our slack currently, and it might be easiest to report it in our workflow rather than customizing the Jenkins jobs.